### DISCUSSION TITLE
Nexus Design Suggestions

### DISCUSSION THREAD

**Discussion by theconnectooor** [icon-project/Nexus#689] [Ideas]
Personally I'm a big fan of the ["Bridge" tab for Synapse.](https://synapseprotocol.com/)

What I like about it is the networks you're bridging between are very clear as well as the tokens you're transferring, and they're all in one screen.

<img width="780" alt="Screen Shot 2022-08-11 at 7 39 52 PM" src="https://user-images.githubusercontent.com/98855020/184264681-f2b23fea-67c0-405b-a00e-78813fc7865b.png">

**Comment by Rook563** [comment]
Hey, hopefully this might work as inspiration for the front end. This is not an easy design to code but just throwing this out there. In the video there's a brief explanation before showing the layout.

https://youtu.be/VHH47ypl82Q

**Reply by CyrusVorwald** [reply]
This is really cool. Can you send me the Nexus BETA logo that pops up around 55 seconds in? Would like to replace our current logo that shows in the top left with it.

**Reply by Rook563** [reply]
Glad you liked it, here's the logo although I don't recommend it to use at a small scale
<img width="2451" alt="nexus-3d-logo" src="https://user-images.githubusercontent.com/111527814/185782213-49d10b35-55c7-43a1-bfbe-19f6593dbd26.png">
.
